packages/core/src/builderProvider/handlers/dynamicContent/placeholder.ts

```diff
@@ -0,0 +1,50 @@
+import { HandlerData } from "@/builderProvider/types/type";
+import { Response } from "@/types/common";
+import { DCPlaceholdersExtra } from "@/types/dynamicContent";
+import { Dictionary } from "@/utils/types";
+
+interface PlaceholderDataHandler extends HandlerData {
+  res: Response<Dictionary<string>>;
+  rej: Response<string>;
+}
+
+function placeholderDataHandler(data: PlaceholderDataHandler) {
+  const { uid, target, res, rej } = data;
+
+  return function placeholderData(event: MessageEvent) {
+    const data = event.data;
+    if (data.ta !== target || data.uid !== uid) {
+      return;
+    }
+
+    try {
+      const action = JSON.parse(data.data);
+      switch (action.type) {
+        case `${target}_dc_placeholder_data_res`: {
```

Comment:
need to be the same like here
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/18303258/83faeedf-2263-4d1b-906e-e346931d3a3f)